defaults/defaults_darwin.go

```diff
@@ -0,0 +1,40 @@
+// +build darwin
+
```

Comment:
nit: the file already has a `_darwin.go` suffix, so this is redundant

Reply:
There is another `defaults.go` file already exits with common constants.